packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -194,9 +194,9 @@ export class LocalRuntimeController extends RuntimeController {
 	};
 
 	onBundleStart(_: BundleStartEvent) {
-		process.on("exit", () => {
-			this.cleanupContainers();
-		});
```

Comment:
This process exit handler was never being removed.
In the tests it resulted in over 70 handlers being added to process.
Apart from potential side-effects, which could well have added to flakiness, it caused lots of warnings to be logged in the test output.